packages/perseus/src/widgets/deprecated-standin/index.tsx

```diff
@@ -22,13 +22,13 @@ class DeprecatedStandin extends React.Component<Props> {
         return noopValidator(1);
     }
 
-    getUserInput: () => UserInput = () => {
+    getUserInput(): UserInput {
```

Comment:
Question, nb: I think you were just removing the arrow functions here, but I noticed this has an empty userInput return. Would you prefer the return type to be `EmptyUserInput` similar to how you changed the other `getUserInput` methods in this PR? I know that type is part of `UserInput`, but was thinking we might want to be consistent with which way we do it.

Reply:
Good catch, fixed